app/modules/asset_groups/resources.py

```diff
@@ -347,11 +347,16 @@ def delete(self, asset_group):
         """
         Delete an Asset_group by ID.
         """
+        _, asset_group_id = asset_group
```

Comment:
can you explain what this line is doing? 
It's a mystery to me.

Reply:
```python
_, asset_group_id = asset_group
```

`asset_group` has to be a tuple or list with 2 items for this to work.  The first item is discarded (by assigning it to `_`) and the second item is assigned to `asset_group_id`.  It can also be written as `asset_group_id = asset_group[1]`.

The reason for this is the decorator for this class:

```python
@api.resolve_object_by_model(AssetGroup, 'asset_group', return_not_found=True)
```

`return_not_found=True` seems to return a tuple of `(asset_group_object, asset_group_id)` and it's stored as `asset_group`.  So `asset_group` isn't actually an asset group object ... A bit confusing...

Reply:
Thanks, I did know that, in fact I wrote the bits that used it above that use it but was having a brain fade :)